src/dodal/devices/ophyd_async_xspress3_mini/xspress3_mini_channel_oa.py

```diff
@@ -0,0 +1,28 @@
+from enum import Enum
+
+from ophyd_async.core import Device
+from ophyd_async.epics.signal import epics_signal_r, epics_signal_rw
+
+
+class TimeSeriesValues(str, Enum):
+    START_VALUE = "Acquire"
+    STOP_VALUE = "Done"
+    UPDATE_VALUE = ""
+
+
+class Xspress3MiniChannel(Device):
+    def __init__(self, prefix: str, name: str = "") -> None:
+        # Define some signals
+        self.update_arrays = epics_signal_rw(float, prefix + "SCAS:TS:TSAcquire")
+
+        self.roi_high_limit = epics_signal_rw(float, prefix + "SCA5_HLM")
+        self.roi_llm = epics_signal_rw(float, prefix + "SCA5_LLM")
+
+        self.time = epics_signal_r(float, prefix + "SCA0:Value_RBV")
+        self.reset_ticks = epics_signal_r(float, prefix + "SCA1:Value_RBV")
+        self.reset_count = epics_signal_r(float, prefix + "SCA2:Value_RBV")
+        self.all_event = epics_signal_r(float, prefix + "SCA3:Value_RBV")
```

Comment:
What does this mean?

Reply:
These are the xpress status so, time it been up, time it been reset etc, not sure we need to collect them for i20-1 but I kept them for historical and being save, I dont want to break something if someone is using them.